README.md

```diff
@@ -1 +1,17 @@
-# github-status
\ No newline at end of file
+# github-status
+
+## Deployment
+Build Docker container with:
+
+`docker build -f ./Dockerfile -t githubstatus .`
+
+Then run it like this:
+
+`docker run --env BOT_TOKEN=<token> --env CHAT_ID=<chat_id> githubstatus`
+
+## Run flake8
+`flake8 --extend-ignore E501 --exclude venv/`
```

Comment:
It seems to me that ignoring the rules is not the best approach:
```
$ flake8 --exclude venv/
./main.py:6:80: E501 line too long (87 > 79 characters)
./settings.py:36:80: E501 line too long (82 > 79 characters)
./settings.py:37:80: E501 line too long (93 > 79 characters)
./settings.py:38:80: E501 line too long (82 > 79 characters)
./settings.py:39:80: E501 line too long (102 > 79 characters)
./settings.py:40:80: E501 line too long (101 > 79 characters)
./test/test_client.py:17:80: E501 line too long (113 > 79 characters)
```

Let's create a [config for flake8](https://flake8.pycqa.org/en/2.5.5/config.html#user-global) and add the string length there (this will already be a rule for the project) and list of excluding directories.